mne/viz/_3d.py

```diff
@@ -1143,3 +1143,107 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
         fig.scene.x_plus_view()
 
     return fig
+
+
+def snapshot_montage(fig, montage, with_points=False):
```

Comment:
does it only apply to ecog?

can you guess from this name that it maps ecog sensors to a pial surface?